engine/servermaster/jobmanager.go

```diff
@@ -51,7 +51,7 @@ type JobManager interface {
 	pb.JobManagerServer
 
 	GetJobMasterForwardAddress(ctx context.Context, jobID string) (string, error)
-	GetJobStatuses(ctx context.Context) (map[frameModel.MasterID]frameModel.MasterStatusCode, error)
+	GetJobStatuses(ctx context.Context) (map[frameModel.MasterID]frameModel.MasterState, error)
```

Comment:
Can we have any plan to rename related variable names or function names like `GetJobStatuses`?

Reply:
To control the size of this pr, we leave it to another pr.